### ISSUE TITLE
Magic Mirror 2 doesn't work in sketch 47

### ISSUE THREAD

**Issue by Sansern** [MagicSketch/MagicMirror#63]
When I Updated Sketch Version 47 and it's not working.
Screenshot Example: https://www.dropbox.com/s/5pgtldtglpdhczz/Screen%20Shot%202017-10-11%20at%2012.50.49%20PM.png?dl=0

**Comment by RvRoo** [comment]
Yes, can confirm. Also doesn't work here. It shows an empty fill on the layer.

**Comment by leeharding** [comment]
Any timeline on getting this to work on 47?
Thanks.

**Comment by michaelpurnell** [comment]
Same magic mirror error here, I would of kept the old version of sketch if I knew this was going to happen.  Could the team please let us know when Magic Mirror will be compatible? Thanks!

**Comment by ghanbak** [comment]
@jamztang 
I'm having the same issue here. Refresh isn't working. 

<img width="1175" alt="screen shot 2017-10-15 at 5 37 56 pm" src="https://user-images.githubusercontent.com/1842556/31589444-a13ed0c6-b1cf-11e7-8e9f-267ef98cd88a.png">

**Comment by leeharding** [comment]
I downgraded to the previous version of sketch. It's very easy, and I don't think there's anything from version 47 that is a deal breaker so I'm happy to wait until this is fixed.

**Comment by jamztang** [comment]
Hi guys, 

We're are currently busying on updating MM3 for compatible with Sketch 47 and 47.1 and we've just pushed some updates few minutes ago.

I've to be honest that for MM2 we're afraid we don't have much resource right now, and back in June we had announced we're retiring a 2.5 year old product, (and I know you're probably not going to be happy to hear that)

We do have a discount code for everyone that come from the MM2 world, just use `LONGTIMEMM2EXTENDED` before selecting a plan in the pro page, then you can unlock basically everything you get in MM2, with even more powerful stuffs like better managing artboards and support for round corners and symbol overrides, we think it's a much better product overall. http://dl.magicsketch.io/mirror3/latest

Regard of that, we should have time soon and we will see what we can do with MM2.

Best
